fdbclient/include/fdbclient/GetEncryptCipherKeys.actor.h

```diff
@@ -319,22 +320,32 @@ Future<TextAndHeaderCipherKeys> getEncryptCipherKeys(Reference<AsyncVar<T> const
                                                      BlobCipherEncryptHeader header,
```

Comment:
We still want a function that takes `BlobCipherEncryptHeaderRef`

Reply:
As it was not used I didn't add as part of this patch; but I think given it is used by multiple inprogress changes, it makes sense to add it here. Let me do that.
Thanks

Reply:
Done.